Apps/W1/Shopify/app/src/Invoicing/Codeunits/ShpfyDraftOrdersAPI.Codeunit.al

```diff
@@ -41,23 +43,21 @@ codeunit 30159 "Shpfy Draft Orders API"
     /// Completes a draft order in shopify by converting it to an order.
     /// </summary>
     /// <param name="DraftOrderId">Draft order id that needs to be completed.</param>
-    /// <param name="NumberOfLines">Maximum amount of possible fulfillment orders.</param>
     /// <returns>Json response of a created order in shopify.</returns>
-    internal procedure CompleteDraftOrder(DraftOrderId: BigInteger; NumberOfLines: Integer): JsonToken
+    internal procedure CompleteDraftOrder(DraftOrderId: BigInteger): JsonToken
     var
         GraphQLType: Enum "Shpfy GraphQL Type";
         Parameters: Dictionary of [Text, Text];
         JResponse: JsonToken;
     begin
         GraphQLType := "Shpfy GraphQL Type"::DraftOrderComplete;
         Parameters.Add('DraftOrderId', Format(DraftOrderId));
-        Parameters.Add('NumberOfOrders', Format(NumberOfLines));
         JResponse := ShpfyCommunicationMgt.ExecuteGraphQL(GraphQLType, Parameters);
         exit(JResponse);
```

Comment:
Would it make more sense for this procedure to now return the OrderId instead of a JSON?

Reply:
We also need order name